source/CIApp.cpp

```diff
@@ -74,14 +74,11 @@ void CoreImpactApp::onStartup() {
  */
 void CoreImpactApp::onShutdown() {
     // save settings file
-    std::shared_ptr<cugl::JsonWriter> _writer = JsonWriter::alloc(Application::getSaveDirectory().append("settings.json"));
-    std::shared_ptr<cugl::JsonValue> _settings = JsonValue::allocObject();
-    _settings->appendValue("PlayerName", _menu.getPlayerName());
-    _settings->appendValue("Volume", _menu.getVolume());
-    _settings->appendValue("MusicOn", _menu.isMusicOn());
-    _settings->appendValue("ParallaxOn", _menu.isParallaxOn());
+    auto _writer = JsonWriter::alloc(Application::getSaveDirectory().append("playersettings.json"));
```

Comment:
Can you put the actual type here? I don't think theres any reason to use auto here b/c you know you're create a JSONWriter